site/en/docs/webstore/cws-enterprise/index.md

```diff
@@ -0,0 +1,143 @@
+This lets you publish an extension that appears on your organization's private Chrome Web Store, so
+any users in your organization can install it. Nobody outside of your organization can see this
+private instance of the Chrome Web Store.
+
+{% Aside %}
+In addition to the privacy and availability benefits of publishing to your Workspace domain, a nice
+side effect is that domain-published extensions usually get through the review process more quickly.
+{% endAside %}
+
+If domain publishing is enabled for your organization, an additional option appears in the
+distribution page of the developer console:
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/SmrqQb3mLZaYj0O3Kdiv.png", alt="Screenshot showing
+the domain publishing option in the developer console", width="643", height="200", class="screenshot" %}
+
+Select this option to publish privately to your domain.
+
+For further instructions on how to publish to your organization's domain, see [Chrome Insider:
+Publishing custom extensions for the enterprise][ent-ext-blog-how].
+
+## Managing domain Chrome Web Store extensions {: #manage-domain-extensions }
+
+There are also features that you don't use directly as an extension developer, but which you should
+be aware of.
+
+**Collections:** Administrators in your organization can curate *collections* of extensions that are
+appropriate to your org. These collections can include both public and privately published
+extensions. These collections are like the ones in the public Chrome Web Store, such as Editor's
+Picks, but appear in your private Chrome Web Store.
+
+The article [Create a Chrome app collection][ent-ext-admin-collections] provides instructions for
+administrators to add collections to your private Chrome Web Store.
+
+**Blocklist and allowlist:** These are options that your organization's administrators can
+use to explicitly control which extensions may or that may not be installed by members of the org.
+
+These and other enterprise extension controls are set by administrators using policies in the
+[extensions atomic policy group][ent-ext-admin-policies].
+
+**Version pinning:** In an enterprise context, it can be a problem when software versions update
+unexpectedly; these changes can necessitate new training and procedures or interfere with
+integrations. For organizations with a private Chrome Web Store enabled, administrators can
+designate specific versions of extensions that users in the organization can install.
+
+## Non-webstore installations {: #side-loading }
+
+There are ways to install enterprise extensions without using the Chrome Web Store, such as those
+described in this section.
+
+**Force install:** Administrators can set policies that automatically install extensions
+for users in the organization. This is typically used for extensions that support workflows and
+other core business operation.
+  
+See the [ExtensionInstallForcelist][ent-ext-admin-forcelist] documentation to learn more about
+this feature.
```

Comment:
We should validate with Fletcher whether ExtensionInstallForceList, ExtensionSettings, or both are the prefered method of controlling this.

Reply:
He did not comment on this in his review, while commenting on adjacent text.